src/app/garfile/craft/index.njk

```diff
@@ -131,8 +131,8 @@
         <a href="/home">{{__('form_exit_without_saving')}}</a>
       </p>
       <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
-		</form>
-	</div>
+	  </div>
+  </form>
```

Comment:
Main tweaks are that the form is now one of the outer elements of the page, as it was actually badly done previously (a two thirds block had the start of the form inside it previously)